### PR TITLE
 LPD-37236 Create playwright test to translate the Rich Text field and check if the translation persists after coming back to the page

### PR DESCRIPTION
**Pull request by rebsilva**
Related issue: https://liferay.atlassian.net/browse/LPD-37236

hi everyone! :cherry_blossom:

this PR has a fix on how the content of the rich text is being rendered related to translations; the changes are on our side (DDM) but the test is on web content, that's why i'm sending this to you as well.
could you please review this commit related to the test? 8b7cf0b4dddb2d09cc1e87ddaa30701d22f1c3f4

thanks a lot and if you have any questions please let me know!

### CODE REVIEW

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

**Comment by rebsilva** [comment]
i saw that the test itself is failing here, i will investigate why since it was passing locally

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 1]
```diff
@@ -1011,6 +1011,68 @@ translationTest(
 	}
 );
 
+translationTest(
+	'Translate the Rich Text field and check if the translation persists after coming back to the page',
```
Comment (DiegoHu97) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:1015]: Here we should use the new `@tag` naming for new tests that we add: 

```javascript
translationTest(
	'Translate the Rich Text field and check if the translation persists after coming back to the page',
	{
		tag: '@LPD-37236'
	},
	async ({journalEditArticlePage, journalPage, page, site}) => {
```

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 2]
```diff
@@ -1011,6 +1011,68 @@ translationTest(
...
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('option', {
+				name: 'Catalan Language: Not Translated',
+			}),
+			trigger: translationButton,
+		});
+
+		await expect(
+			journalEditArticlePage.journalPage.articleContentTextBox.getByText(
```
Comment (DiegoHu97) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:1069]: The same here,just use `journalPage.articleContentTextBox.getByText...`

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 3]
```diff
@@ -1011,6 +1011,68 @@ translationTest(
...
+			page,
+			`Success:${title} was created successfully.`
+		);
+
+		await page.getByRole('link', {name: title}).click();
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('option', {
+				name: 'Catalan Language: Not Translated',
```
Comment (DiegoHu97) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:1063]: here the name should be `{ name: 'Catalan Language: Translating 1/2' }` as teh catalan language has the rich text field translated, this is the reason the test fails

**Comment by DiegoHu97** [comment]
Hey @rebsilva ! i just left you a couple of comments, I just added the solution for the failing test too, thanks :smile:

**Comment by rebsilva** [comment]
thanks a lot @DiegoHu97!! i'll take a look now :blush:

**Comment by rebsilva** [comment]
hi @DiegoHu97! i've updated the PR with the changes, could you please review again? thanks a lot for all the help!

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

**Comment by DiegoHu97** [comment]
ci:report:27663074

**Comment by DiegoHu97** [comment]
hi @rebsilva there are no failing tests related, do you want me to forward or you forward it in your repo?

**Pull request closed without merging**